src/test/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApiSpek.kt

```diff
@@ -182,6 +183,47 @@ class AktivitetskravApiSpek : Spek({
                             )
                         }
                     }
+
+                    it("Returns aktivitetskrav with stoppunkt after cutoff") {
+                        val cutoffDate = externalMockEnvironment.environment.arenaCutoff
+                        val aktivitetskravBeforeCutoff = Aktivitetskrav.ny(
```

Comment:
Kan evt sette stoppunkt på denne og den under til en dato _før_ cutoff (ikke lik) så navnet stemmer. Evt en som er før og en som er lik for å sjekke at det er after som gjelder :)

Reply:
Bra sett, både før cutoff og samme dato som cutoff skal fjernes fra listen, så greit å sjekke begge deler 👍